backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -119,6 +126,16 @@
         return resources.createEndpoint(endpoint);
     }
 
+    @POST
+    @Path("/system/email_subscription")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    @RolesAllowed(RbacIdentityProvider.RBAC_READ_INTEGRATIONS_ENDPOINTS)
```

Comment:
`READ` or `WRITE`?

Reply:
I think it's debatable.
Only `READ` permissions are fine, we want to get the default email subscription endpoint - If in the process it has to create one, its fine IMO.

Reply:
I agree and in the meantime it's weird to have a `POST` operation with `READ` requirement 😄

Reply:
Let's stick with READ, it makes sense.

Reply:
Think of an API user with `READ` permissions that wants to get the `EmailSubscription` to see the history.